.github/workflows/deploy-github-page.yml

```diff
@@ -70,12 +70,11 @@ jobs:
           jupyter lite build \
               --XeusAddon.prefix=${{ env.PREFIX }} \
               --XeusAddon.mounts=${{ env.PREFIX }}/share/jupyter/kernels/xr/resources:/share/jupyter/kernels/xr/resources \
+              --contents README.md \
+              --contents notebooks/examples.ipynb \
+              --contents notebooks/xeus-r.ipynb \
```

Comment:
Maybe we should not include xeus-r.ipynb just yet, because of the plotting issues

Reply:
Sounds good, will remove it. Thanks!